### PR TITLE
Feature/check gatk sample map paths

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- See commit msg below

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [ ] Code review
- [ ] New code is executed and covered by tests
- [ ] Tests pass

### CODE REVIEW

File: lib/MIP/Gatk.pm [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        ## Path exists
+        next LINE if ( -e $file_path );
+
+        $log->fatal( q{The supplied file path: }
+              . $file_path
+              . q{ from sample map file: }
+              . $sample_map_path
+              . q{ does not exist} );
+        exit 1;
```
Comment (jemten) [inline lib/MIP/Gatk.pm:92]: You could bundle all the missing paths in an array and then later loop over the array to display all the missing paths.

File: lib/MIP/Gatk.pm [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    foreach my $line (@lines) {
+
+        ## Get sample and file path (and check proper format)
+        my ( $sample, $file_path, $unexpected_data ) =
+          split $TAB, $line, $FIELD_COUNTER + 1;
+
+        ## Make sure that we get what we expect
+        if ( defined $unexpected_data ) {
+
+            carp q{Unexpected trailing garbage at end of line '} . $line . q{':},
```
Comment (jemten) [inline lib/MIP/Gatk.pm:80]: logcarp?

File: lib/MIP/Gatk.pm (hunk 1 above)
Comment (henrikstranneheim) [reply lib/MIP/Gatk.pm]: Implemented

File: lib/MIP/Gatk.pm (hunk 2 above)
Comment (henrikstranneheim) [reply lib/MIP/Gatk.pm]: Done

**Pull request merged**